crates/types/src/config/mod.rs

```diff
@@ -231,4 +232,16 @@ impl Configuration {
     pub fn dump(&self) -> Result<String, GenericError> {
         Ok(toml::to_string_pretty(self)?)
     }
+
+    /// Getter for the default cluster configuration
+    pub fn default_cluster_configuration(&self) -> ClusterConfigurationSeed {
+        match &self.common.default_cluster_configuration {
+            Some(custom_config) => custom_config.clone(),
```

Comment:
Maybe check if one of the deprecated options is configured and log a message telling users that these settings are no longer respected and should be migrated.